hbase-mapreduce/src/main/java/org/apache/hadoop/hbase/mapreduce/WALPlayer.java

```diff
@@ -172,6 +180,52 @@ public void map(WALKey key, WALEdit value, Context context) throws IOException {
           ExtendedCell lastCell = null;
           for (ExtendedCell cell : WALEditInternalHelper.getExtendedCells(value)) {
             context.getCounter(Counter.CELLS_READ).increment(1);
+
+            if (CellUtil.matchingQualifier(cell, WALEdit.BULK_LOAD)) {
```

Comment:
I believe the processing of bulkloaded files can be simplified, and we could reduce the log level from INFO to DEBUG or TRACE in some cases.